allennlp/commands/train.py

```diff
@@ -311,6 +341,7 @@ def _train_worker(
     file_friendly_logging: bool = False,
     include_package: List[str] = None,
     batch_weight_key: str = "",
+    dry_run: bool = False,
```

Comment:
In general add new args to the end of the parameter list.

Reply:
I put it here because the rest are for distributed training.

Reply:
Fine here given that the function is internal, but in general that sort of grouping wouldn't take precedence over keeping code working.